### ISSUE TITLE
Decreased dihydrotestosterone serum concentration

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#6886]
**Preferred term label:**
Decreased dihydrotestosterone serum concentration

**Definition (free text, please give PubMed ID)**
Decrease of dihydrotestosterone level in serum 
20301592[PMID]; 3312397[PMID]

**Parent term (use hpo.jax.org/app)**
Decreased circulating androgen level (HP:0030349)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Congenital adrenal hyperplasia due to cytochrome P450 oxidoreductase deficiency (ORPHA 95699)

**Comment by e-lbtn** [comment]
A mistake was made on the referencement in the definition part, the PMID are 31888681 and 20301592. 
All my apologies !

**Comment by pnrobinson** [comment]
thanks, added as HP_0033810
